### PR TITLE
fix non-Bearer auth scheme access bug.

### PR DESCRIPTION
**Pull request by vikitikitavi**

### CODE REVIEW

File: aiohttp_jwt/middleware.py [hunk 1]
```diff
@@ -59,10 +59,13 @@ def JWTMiddleware(
...
-                    if credentials_required and not re.match('Bearer', scheme):
-                        raise aiohttp.web.HTTPForbidden(
-                            reason='Invalid token scheme',
-                        )
+                    if not re.match('Bearer', scheme):
+                        if credentials_required:
+                            raise aiohttp.web.HTTPForbidden(
+                                reason='Invalid token scheme',
+                            )
+                        else:
```
Comment (hzlmn) [inline aiohttp_jwt/middleware.py]:
```python

Comment (hzlmn) [inline aiohttp_jwt/middleware.py]: Lets exit earlier here `return await handler(request)`

**Comment by hzlmn** [comment]
Anyway, i can cherry pick this pr if you are busy, just ping me.

**Comment by hzlmn** [comment]
Thanks

**Comment by vikitikitavi** [comment]
sorry for delay

**Comment by hzlmn** [comment]
No problems, there is no time deadlines 😃

**Pull request merged**